src/rmkit/ui/keyboard.cpy

```diff
@@ -241,3 +242,7 @@ namespace ui:
     void switch_mode(int mode):
       // TODO
       pass
+
+  ;
```

Comment:
There were a couple files where it looked like okp wasn't adding the semicolon after the last class. Maybe when there are multiple classes in the file?